anime_downloader/sites/yify.py

```diff
@@ -0,0 +1,45 @@
+import requests, json, os
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+from bs4 import BeautifulSoup
+from io import StringIO
+
+class Yify(Anime, sitename = 'yify'):
+    sitename = 'yify'
+    url = f'https://{sitename}.mx'
```

Comment:
yts.mx and yify.mx, although seemingly affiliated are two different sites - could you change the title of your PR to reflect that?